datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/deser/LocalDateDeserTest.java

```diff
@@ -476,7 +478,7 @@ public void testLenientDeserializeFromNumberInt() throws Exception {
         mapper.configOverride(LocalDate.class)
                         .setFormat(JsonFormat.Value.forShape(JsonFormat.Shape.NUMBER_INT));
 
-        assertEquals("The value is not correct.", LocalDate.of(1970, Month.MAY, 04),
+        assertEquals("The value is not correct.", LocalDate.of(1970, Month.MAY, 4),
```

Comment:
Odd, I wonder why it used to be 04 vs. 4.. anyway, this cleanup looks fine to me.

Reply:
Maybe seemed more readable by original author? (I didn't write it)